build.gradle

```diff
@@ -46,30 +45,37 @@ subprojects {
         jcenter()
     }
 
+    // FIXME: This picks up non-Java files by default, failing to parse them.
+    // Need to configure this more carefully.
+    tasks.withType(Javadoc).all {
+        enabled = false
+    }
+
     configurations.all {
         resolutionStrategy.force Deps.jsr305
     }
 
     task allclean {
     }
 
-    afterEvaluate { project ->
-      allclean.dependsOn(project.tasks.matching {it.name == 'clean'})
-
-        if (project.tasks.matching { it.name == 'assembleRelease'}) {
-            def jarList = new ArrayList()
-            task setJarList(dependsOn: assembleRelease) << {
-                configurations.compile.each {
-                    jarList.add(it)
-                }
-            }
-
-            task copyDeps(dependsOn: setJarList, type: Copy) {
-                from jarList
-                into 'build/external'
-            }
-        }
-    }
+// TODO: Figure out why this breaks (and what it does).
```

Comment:
@oprisnik I'm not quite sure why this one breaks but also what it's there to do in the first place. 😅

Given that it's only run on cleanup, I suppose it's not super critical but would still be good to understand what the intent is.

It's failing now with this:

```
FAILURE: Build failed with an exception.

* Where:
Build file '/Users/realpassy/Projects/java/fresco/build.gradle' line: 65

* What went wrong:
A problem occurred configuring project ':animated-base'.
> Could not create task ':animated-base:uploadArchives'.
   > Task with name 'publishAllPublicationsToMavenCentralRepository' not found in project ':animated-base'.
```

Reply:
I don't think we need this any more. Seems to be legacy code for Eclipse support :)